project.clj

```diff
@@ -7,22 +7,38 @@
 
   :pedantic? :abort
 
-  :dependencies [[org.clojure/clojure "1.6.0"]
-                 [clj-time "0.7.0"]
+  :dependencies [[org.clojure/clojure "1.7.0"]
+                 [clj-time "0.10.0"]
+                 [commons-codec "1.9"]
+                 [org.clojure/tools.macro "0.1.5"]
```

Comment:
I presume some of these are to resolve transitive dependency version conflicts w/lein pedantic?  If so can we try to go ahead and add the leading/trailing comments as per https://confluence.puppetlabs.com/pages/viewpage.action?pageId=48956980